requirements.txt

```
@@ -3,8 +3,8 @@ bioblend>=1.0.0
 click!=8.0.2
 cwltool>=1.0.20191225192155
 ephemeris>=0.10.3
-galaxy-tool-util>=23.0,<23.1
-galaxy-util[template]>=23.0,<23.1
+galaxy-tool-util>=23.0,<24.0
+galaxy-util[template]>=23.0,<24.0
```

I don't think we have officially ruled out a 23.2 release.

```suggestion
galaxy-tool-util>=23.0,<23.2
galaxy-util[template]>=23.0,<23.2
```